seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-juicefs-e2e/src/test/java/org/apache/seatunnel/e2e/connector/file/juicefs/JuicefsFileIT.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.e2e.connector.file.juicefs;
+
+import org.apache.seatunnel.e2e.common.TestResource;
+import org.apache.seatunnel.e2e.common.TestSuiteBase;
+import org.apache.seatunnel.e2e.common.container.ContainerExtendedFactory;
+import org.apache.seatunnel.e2e.common.container.TestContainer;
+
+import org.apache.seatunnel.e2e.common.junit.TestContainerExtension;
+import org.junit.jupiter.api.*;
+import org.testcontainers.containers.Container;
+
+import java.io.IOException;
+
+@Disabled("Please testing it in your local environment with juicefs conf")
```

Comment:
why not start juicefs on docker https://juicefs.com/docs/community/juicefs_on_docker/